### PR TITLE
Control Flow - Train

### PR DESCRIPTION
**Pull request by prateek-mehra**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [ ] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [ ] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [ ] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [ ] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Comment by vinthaivendhan10x** [comment]
@satya-9 kindly Review

**Review by satya-9 — APPROVED** [review]

File: solutions/control flow/train.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let totalCarsAndCarNumberFromFront = readLine().split(" ");
```
Comment (sailesh-ramayanam) [inline solutions/control flow/train.js:14]: Extremely long variable names are counter-productive to readability. Please rename this to something like `inputs`.

File: solutions/control flow/train.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+totalCars = parseInt(totalCarsAndCarNumberFromFront[0]);
+carNumberFromFront = parseInt(totalCarsAndCarNumberFromFront[1]);
+
+carNumberFromBack = totalCars - carNumberFromFront + 1;
```
Comment (sailesh-ramayanam) [inline solutions/control flow/train.js:19]: Please use `let` to declare variables.

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]
@prateek-mehra 
Please rename the folder to `control-flow`. Other comments are below.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**